.travis.yml

```diff
@@ -1,71 +1,30 @@
-branches:
-  only:
-    - master
-    - develop
```

Comment:
Can we disable "Build pushed pull requests" in Travis configs instead?
![Screenshot 2020-06-17 at 10 58 20](https://user-images.githubusercontent.com/9007851/84842883-b14f6500-b089-11ea-9328-91e94c2a481e.png)